### DISCUSSION TITLE
Abstract class vs interface

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#13] [General]
# Abstract class vs interface

In C# we have abstract classes and we also have interfaces. Both seem very similiar but they are fundamentally different concepts.
Therefore let's discover what the difference is, what the overlap is and when to use what.

https://steven-giesel.com/blogPost/3992a289-4cc9-460c-b484-0cc137cccafa

**Comment by usamekilicbay** [comment]
I've read lots of articles and watches many tutorials to understand "Why I need interfaces? I can use abstract classes instead of interfaces as well." Finally I got the point. Thank you for detailed explanation, clean examples and reasons Steven!
